lighthouse-core/gather/gather-runner.js

```diff
@@ -410,7 +430,8 @@ class GatherRunner {
         await driver.setThrottling(options.settings, passConfig);
         await GatherRunner.beforePass(passContext, gathererResults);
         await GatherRunner.pass(passContext, gathererResults);
-        const passData = await GatherRunner.afterPass(passContext, gathererResults);
+        const [passData, pageLoadError] =
```

Comment:
@brendankenny @patrickhulce either of you have an idea on how else to deliver this error?

see also the L462 here

Reply:
`pageLoadError.fatal = true; throw pageLoadError;` :P